plugins/woocommerce-admin/package.json

```diff
@@ -10,17 +10,20 @@
 	},
 	"scripts": {
 		"analyze": "cross-env NODE_ENV=production ANALYZE=true webpack",
-		"build": "node ./node_modules/require-turbo && pnpm run clean && WC_ADMIN_PHASE=core pnpm run build:feature-config && cross-env NODE_ENV=production WC_ADMIN_PHASE=core webpack",
+		"build": "pnpm -w exec turbo run turbo:build --filter=$npm_package_name -- --",
+		"test": "pnpm -w exec turbo run turbo:test --filter=$npm_package_name -- --",
+		"turbo:build": "pnpm run clean && WC_ADMIN_PHASE=core pnpm run build:feature-config && cross-env NODE_ENV=production WC_ADMIN_PHASE=core webpack",
+		"lint": "pnpm run lint:js && pnpm run lint:css",
+		"turbo:test": "pnpm run test:client",
 		"build:feature-config": "php ../woocommerce/bin/generate-feature-config.php",
-		"build:packages": "cross-env NODE_ENV=production pnpm -w exec turbo run build --filter='./packages/js/*'",
+		"build:packages": "cross-env NODE_ENV=production pnpm -w run build --filter='./packages/js/*'",
 		"clean": "rimraf ../woocommerce/assets/client/admin/*",
 		"client:watch": "cross-env WC_ADMIN_PHASE=development pnpm run build:feature-config && cross-env WC_ADMIN_PHASE=development webpack --watch",
 		"create-hook-reference": "node ./bin/hook-reference/index.js",
 		"create-wc-extension": "node ./bin/starter-pack/starter-pack.js",
-		"dev": "cross-env WC_ADMIN_PHASE=development pnpm run build:feature-config && cross-env WC_ADMIN_PHASE=development pnpm -w exec turbo run build --filter='./packages/js/*' && cross-env WC_ADMIN_PHASE=development webpack",
+		"dev": "cross-env WC_ADMIN_PHASE=development pnpm run build:feature-config && cross-env WC_ADMIN_PHASE=development pnpm -w run build --filter='./packages/js/*' && cross-env WC_ADMIN_PHASE=development webpack",
```

Comment:
This script feels like it's doing a _lot_ that it doesn't need to. 

* Can we add `build:feature-config` to `"turbo:build"`? Would you want it to build the feature config whenever it builds the command?
* We don't need to run `build` on all of the JS packages, right? They're dependencies of `woocommerce/client/internal` and Turbo will build them for us, right?
* If the above is all true, can we replace this with `cross-env WC_ADMIN_PHASE=development pnpm run build && cross-env WC_ADMIN_PHASE=development webpack` or something similar?

Reply:
On top of that, do we need `cross-env` this many times? If you set it at the beginning of the script, does it get set for everything after? That seems like it would be the case, right?

Reply:
@ObliviousHarmony I am sure this script could be improved, but is that really under the purview of this PR?

Reply:
That's fair, I've added it as a comment to the PR to https://github.com/woocommerce/woocommerce/issues/33948